mysql_servers.json

```diff
@@ -1,5 +1,66 @@
 {
   "apps": [
+    {
+      "name": "MySQL server PORT 3306",
```

Comment:
Did you mean to re-order all the services?

Reply:
yes, since some of the docker process take longer to start I moved them up